cosmpy/aerial/config.py

```diff
@@ -125,8 +140,8 @@ def fetchai_mainnet(cls) -> "NetworkConfig":
             chain_id="fetchhub-4",
             url="grpc+https://grpc-fetchhub.fetch.ai",
             fee_minimum_gas_price=0,
```

Comment:
Note: We have been considering setting this default minimum gas price here to non-zero value (the same value as for testnet). Even though this is not enforceable, practically speaking, this is what we want - if nothing else, then at least  defaulting to non-zero Tx gas price value will make it less straightforward for users to use zero Tx fee.

Reply:
For now (for ASI umgrade release), please ignore my comment above. We will touch on thsese kind of things in later releases.

Reply:
Note: In this sense (= not now), we shall also extend the `NetworkConfig` class with address prefix data member.